tests/spec/permalink.js

```diff
@@ -61,7 +61,7 @@ describe('streamhub-permalink/permalink', function () {
             expect(pl._warehouse[key]).toBe(content);
         });
         
-        xit('throws if you try to ._set() without a key or item', function () {
+        it('throws if you try to ._set() without a key or item', function () {
```

Comment:
I remember disabling this because it didn't pass. I think it's stupid to have a test for a '_private' method...